docs/content/docs/tasks/restart-application-deployment/_index.md

```diff
@@ -0,0 +1,165 @@
+
+```shell
+$ kubectl get keptnappversions.lifecycle.keptn.sh -n restartable-apps -owide
+NAME                   APPNAME        VERSION   PHASE               PREDEPLOYMENTSTATUS   PREDEPLOYMENTEVALUATIONSTATUS   WORKLOADOVERALLSTATUS   POSTDEPLOYMENTSTATUS   POSTDEPLOYMENTEVALUATIONSTATUS
+podtato-head-0.1.1-1   podtato-head   0.1.1     AppPreDeployTasks   Failed                Deprecated                      Deprecated              Deprecated             Deprecated
+```
+
+Now, to fix the deployment of this application is to first fix the task that has failed earlier. To do so, edit the
+`pre-deployment-check` `KeptnTaskDefinition` to the following (`kubectl -n restartable-apps edit keptntaskdefinitions.lifecycle.keptn.sh pre-deployment-check`):
+
+```yaml
+apiVersion: lifecycle.keptn.sh/v1alpha2
+kind: KeptnTaskDefinition
+metadata:
+  name: pre-deployment-check
+  namespace: restartable-apps
+spec:
+  function:
+    inline:
+      code: |
+        console.error("Success")
+```
+
+After we have done that, we can restart the deployment of our `KeptnApplication` by increasing the `spec.revision` field by one 
+(`kubectl -n restartable-apps edit keptnapps.lifecycle.keptn.sh podtato-head`):
+
+```yaml
+apiVersion: lifecycle.keptn.sh/v1alpha2
+kind: KeptnApp
+metadata:
+  name: podtato-head
+  namespace: restartable-apps
+spec:
+  version: "0.1.1"
+  revision: 2 # Increased this value from 1 to 2
+  workloads:
+    - name: podtato-head-entry
+      version: "0.1.2"
+  preDeploymentTasks:
+    - pre-deployment-check
+```
+
+After those changes have been made, you'll notice a new revision of the `podtato-head` `KeptnAppVersion`: 
+
+```shell
+$ kubectl get keptnappversions.lifecycle.keptn.sh -n restartable-apps       
+NAME                   APPNAME        VERSION   PHASE
+podtato-head-0.1.1-1   podtato-head   0.1.1     AppPreDeployTasks
+podtato-head-0.1.1-2   podtato-head   0.1.1     AppDeploy
+```
+
+As you will see, the newly created revision `podtato-head-0.1.1-2` has made it beyond the preDeployment check phase and has reached its `AppDeployPhase`.
+
+You can also verify the execution of the `pre-deployment-check` by retrieving the list of `KeptnTasks` in the `restartable-apps` namespace:
+
+```shell
+$ kubectl get keptntasks.lifecycle.keptn.sh -n restartable-apps
+NAME                             APPNAME        APPVERSION   WORKLOADNAME   WORKLOADVERSION   JOB NAME                              STATUS
+pre-pre-deployment-check-49827   podtato-head   0.1.1                                         klc-pre-pre-deployment-check--77601   Failed
+pre-pre-deployment-check-65056   podtato-head   0.1.1                                         klc-pre-pre-deployment-check--57313   Succeeded
+```
+
+You'll notice that for both the `KeptnAppVersions` and `KeptnTasks` the previous failed instances are still available, as this might be useful historical data to keep track of 
+what went wrong during earlier deployment attempts.
```

Comment:
```suggestion
You will notice that for both the `KeptnAppVersions` and `KeptnTasks` the previous failed instances are still available, as this might be useful historical data to keep track of 
what went wrong during earlier deployment attempts.
```